esiclient/v1/node_network.py

```diff
@@ -264,21 +264,40 @@ def take_action(self, parsed_args):
         ironic_client = self.app.client_manager.baremetal
         neutron_client = self.app.client_manager.network
 
-        node = None
-        port = None
+        current_node = ironic_client.node.get(node_uuid)
 
-        with concurrent.futures.ThreadPoolExecutor() as executor:
-            f1 = executor.submit(ironic_client.node.get, node_uuid)
-            f2 = executor.submit(neutron_client.find_port, port_uuid)
-            node = f1.result()
-            port = f2.result()
-
-        if not port:
+        if current_node is None:
             raise exceptions.CommandError(
-                "ERROR: Port {1} not attached to node {0}".format(
-                    node.name, port_uuid))
+                "ERROR: The node {0} is not available".format(
+                    node_uuid))
+
+        if port_uuid:
+            port = neutron_client.find_port(port_uuid)
+            if not port:
+                raise exceptions.CommandError(
+                    "ERROR: Port {1} not attached to node {0}".format(
+                        current_node.name, port_uuid))
+        else:
+            ports = ironic_client.port.list(node=node_uuid, detail=True)
```

Comment:
Can you rename this to `baremetal_ports` or `bm_ports`? I think it's important for someone else looking at the code to be able to distinguish between baremetal ports and neutron ports.